pkg/k8shandler/cluster.go

```diff
@@ -59,14 +59,18 @@ func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateElasticsearchClu
 
 		if _, ok := containsNodeTypeInterface(upgradeInProgressNode, scheduledUpgradeNodes); ok {
 			logrus.Debugf("Continuing update for %v", upgradeInProgressNode.name())
-			upgradeInProgressNode.update(nodeStatus)
+			if err := upgradeInProgressNode.update(nodeStatus); err != nil {
```

Comment:
i don't think we want to fail out here because we want to make sure we update the node state every time...

Reply:
Fixed by printing the error instead of returning it.